pkg/kdump/kdump-view.jsx

```diff
@@ -263,12 +264,33 @@ export class KdumpPage extends React.Component {
     }
 
     handleTestSettingsClick() {
+        // if we have multiple targets defined, the config is invalid
+        const target = this.props.kdumpStatus.target;
+        let verifyMessage;
+        if (!target.multipleTargets) {
+            if (target.type === "local") {
+                verifyMessage = fmt_to_fragments(
+                    ' ' + _("To verify whether the crash was successful check $0 location for kernel crash dump(vmcore)."),
+                    <span className="ct-monospace">{target.path || "/var/crash"}</span>);
+            } else if (target.type === "ssh" || target.type === "nfs") {
+                verifyMessage = fmt_to_fragments(
+                    ' ' + _("To verify whether the crash was successful check $0 location at $1 for kernel crash dump(vmcore)."),
+                    <span className="ct-monospace">{target.path || "/var/crash"}</span>,
+                    <span className="ct-monospace">{target.type === "ssh" ? target.server : `${target.server}:${target.export}`}</span>);
+            }
+        }
+
         // open a dialog to confirm crashing the kernel to test the settings - then do it
         const dialogProps = {
             title: _("Test kdump settings"),
-            body: (
-                <span>{_("This will test kdump settings by crashing the kernel and thereby the system. Depending on the settings, the system may not automatically reboot and the process may take a while.")}</span>
-            )
+            body: (<>
+                <span>
+                    {_("This will test kdump settings by crashing the kernel and thereby the system. Depending on the settings, the system may not automatically reboot and the process may take a while.")}
+                </span>
+                {verifyMessage && <span>
+                    {verifyMessage}
+                </span>}
```

Comment:
These 7 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19107-20230718-124527-bf8cfcb1-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)